woof-code/rootfs-petbuilds/palemoon/petbuild

```diff
@@ -1,9 +1,9 @@
 download() {
-    [ -f palemoon-31.3.1.linux-x86_64-gtk3.tar.xz ] || wget -t 3 -T 60 https://linux.palemoon.org/datastore/release/palemoon-31.3.1.linux-x86_64-gtk3.tar.xz
+    [ -f palemoon-31.4.0.linux-x86_64-gtk3.tar.xz ] || wget -t 3 -T 60 http://linux.palemoon.org/datastore/release/palemoon-31.4.0.linux-x86_64-gtk3.tar.xz
```

Comment:
Why not HTTPS?

Reply:
Oh, sorry. http://linux.palemoon.org/download/mainline/ gave same link (provided officially by palemoon). I am updating it now. I'll also want some consultation with palemoon download links on its forum. I had a failed build just now because palemoon site updated its link. I think it has some separate sub-doomain for older downloads.

By that time, this PR would keep things 'safe'.

Reply:
Updating to https now.